command/fileserver/fileserver.go

```diff
@@ -0,0 +1,113 @@
+package fileserver
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+
+	"github.com/smallstep/cli/errs"
+
+	"github.com/smallstep/cli/command"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	cmd := cli.Command{
+		Name:   "fileserver",
+		Hidden: true,
+		Action: command.ActionFunc(fileServerAction),
+		Usage:  "start an HTTP(S) server serving the contents of a path",
```

Comment:
maybe say "contents of a directory" since you're calling the arg <dir>

Reply:
👍

Reply:
the usage would be too long to keep it in one line, leaving it as path